File: Bossinfo.Module.Hospital/orderregresultdetail.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Bossinfo.Module.Hospital
{
    public class orderregresultdetail
    {
        [Key]
        [Column(Order = 0)]
        [DatabaseGenerated(DatabaseGeneratedOption.None)]
        public long ID { get; set; }

        [Key]
        [Column(Order = 1)]
        [DatabaseGenerated(DatabaseGeneratedOption.None)]
        public long OrderRegResultID { get; set; }

        [StringLength(100)]
        public string ItemID { get; set; }

        [StringLength(200)]
        public string ItemName { get; set; }

        [StringLength(20)]
        public string ResultNoteOriginalValue { get; set; }

        [StringLength(200)]
        public string Result { get; set; }

        [StringLength(5)]
        public string ResultNote { get; set; }

        [StringLength(500)]
        public string ReferenceVauleDescription { get; set; }

        [Key]
        [Column(Order = 2)]
        [DatabaseGenerated(DatabaseGeneratedOption.None)]
        public int Sort { get; set; }

        [Key]
        [Column(Order = 3)]
        public bool IsDelete { get; set; }

        [Column(TypeName = "datetime2")]
        public DateTime? CreatedTime { get; set; }

        public long? CreatedBy { get; set; }

        [Column(TypeName = "datetime2")]
        public DateTime? LastModifiedTime { get; set; }

        public long? LastModifiedBy { get; set; }
    }
}

File: Bossinfo.Models.Hospital/orderregresultdetail.ts

 //改NameSpace就好
 
    import { BossApiModel, Key } from '@boss/decorator/BossApiModel';
    import { I18nNamespace } from '@boss/decorator/i18n/i18nNs';
    @BossApiModel('api/orderregresultdetail')
    @I18nNamespace('Model.EDI.orderregresultdetail')
    export class orderregresultdetail {
        
        @Key()// Column(Order = 0)// DatabaseGenerated(System.ComponentModel.DataAnnotations.Schema.DatabaseGeneratedOption.None)
        public ID: number;
        @Key()// Column(Order = 1)// DatabaseGenerated(System.ComponentModel.DataAnnotations.Schema.DatabaseGeneratedOption.None)
        public OrderRegResultID: number;
        // StringLength(100)
        public ItemID: string;
        // StringLength(200)
        public ItemName: string;
        // StringLength(20)
        public ResultNoteOriginalValue: string;
        // StringLength(200)
        public Result: string;
        // StringLength(5)
        public ResultNote: string;
        // StringLength(500)
        public ReferenceVauleDescription: string;
        @Key()// Column(Order = 2)// DatabaseGenerated(System.ComponentModel.DataAnnotations.Schema.DatabaseGeneratedOption.None)
        public Sort: number;
        @Key()// Column(Order = 3)
        public IsDelete: boolean;
        // Column(TypeName = "datetime2")
        public CreatedTime: Date;
        
        public CreatedBy: number;
        // Column(TypeName = "datetime2")
        public LastModifiedTime: Date;
        
        public LastModifiedBy: number;
    }
    
    /*
    "orderregresultdetail": {
        
        "ID": "ID",
        "OrderRegResultID": "OrderRegResultID",
        "ItemID": "ItemID",
        "ItemName": "ItemName",
        "ResultNoteOriginalValue": "ResultNoteOriginalValue",
        "Result": "Result",
        "ResultNote": "ResultNote",
        "ReferenceVauleDescription": "ReferenceVauleDescription",
        "Sort": "Sort",
        "IsDelete": "IsDelete",
        "CreatedTime": "CreatedTime",
        "CreatedBy": "CreatedBy",
        "LastModifiedTime": "LastModifiedTime",
        "LastModifiedBy": "LastModifiedBy"
    }
    */
    
